### PR TITLE
Add configuration for Kotlin static code analyser  

### LINKED ISSUE #6: Add static code analyser to project.
**Summary.**
Add a static code analyser for linting Kotlin files.

### PR DESCRIPTION
**Pull request by juanpa097**
### Summary
Adde [this plugin](https://github.com/gantsign/ktlint-maven-plugin) to maven. The plugin uses ktlint to analyse code.

### Test Plan
Run `mvn com.github.gantsign.maven:ktlint-maven-plugin:1.4.1:check` to check code.
Run `mvn com.github.gantsign.maven:ktlint-maven-plugin:1.4.1:format` to fix lints.

### Related Issues
closes #6
Screenshot 2020-02-18 at 10 24 38 pm

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (johan-smc) [inline .idea/compiler.xml]: He is a good idea to have a version of the IntelliJ configurations?

File: pom.xml [hunk 2]
```diff
@@ -108,6 +108,19 @@
                     <compilerArgs>--enable-preview</compilerArgs>
```
Comment (johan-smc) [inline pom.xml]: I believe that this line needs to remove to run the build, no?

File: .idea/compiler.xml (hunk 1 above)
Comment (juanpa097) [reply .idea/compiler.xml]: Should we remove this file then? 😰

Comment (johan-smc) [reply .idea/compiler.xml]: No, keep the changes

**Pull request merged**